src/redux/actions/category.action.js

```diff
@@ -0,0 +1,15 @@
+import {GET_CATEGORY, GET_CATEGORY_SUCCESS} from '../constants';
+
+export function getCategory(payload){
+  return {
+    type: GET_CATEGORY,
+    payload: payload
```

Comment:
```suggestion
    payload,
```
cú pháp ngắn gọn tương đương nhé c